Powershell/InstallHuntress.powershellv2.ps1

```diff
@@ -552,6 +559,28 @@ function uninstallHuntress {
     }
 }
 
+
+
+
+# grab the currently installed agent version AB
+function getAgentVersion {
+    $exeAgentPath = Join-Path (getAgentPath) "HuntressAgent.exe"
+    $agentVersion = (Get-Item $exeAgentPath).VersionInfo.FileVersion
+    LogMessage "Agent version $($agentVersion) found"
+    return $agentVersion
+}
+
+# ensure all the Huntress services are running and set them to delayed start AB
+function repairAgent {
+    sc.exe stop HuntressAgent 
+    sc.exe stop HuntressUpdater 
+    sc.exe config HuntressAgent start= delayed-auto 
```

Comment:
@Alan-Huntress we don't want to move autostart to delayed in the script - the reason is because we are going to be improving the agent startup in the future so delayed is no longer needed. For this exercise, just bringing the updater back online will be enough to make the agent update and then it will be good.